packages/ui/src/components/Button.tsx

```diff
@@ -19,21 +21,24 @@ export const Button: FC<Props> = ({
   theme = "default",
   disabled,
   type = "button",
+  size = "medium",
   children,
 }) => {
   return (
     <button
       className={clsx(
-        "rounded-md h-8 px-4 border text-sm font-medium flex items-center justify-center space-x-1",
+        "border text-xs font-medium flex items-center justify-center space-x-1",
```

Comment:
Is `text-xs` here (and not in `size=small` mode) intentional?
Looks a bit too small to me; smaller than typical github buttons.

<img width="563" alt="Captura de pantalla 2023-06-28 a la(s) 20 31 12" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/74d70fc1-87ec-4a80-a554-f03e5a40c628">

Reply:
you're right, my bad